binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,31 @@
+class BST:
+    def __init__(self, val=None):
+        self.left = None
+        self.right = None
+        self.val = val
+
+    def __str__(self):
+        return "[%s, %s, %s]" % (self.left, str(self.val), self.right)
+
+    def isEmpty(self):
+        return self.left == self.right == self.val == None
+
+    def insert(self, val):
+        if self.isEmpty():
+            self.val = val
+        elif val < self.val:
+            if self.left is None:
+                self.left = BST(val)
+            else:
+                self.left.insert(val)
+        else:
+            if self.right is None:
+                self.right = BST(val)
+            else:
+                self.right.insert(val)
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7yepr_3t/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmp7yepr_3t/binary_search_tree/BinarySearchTree.py
@@ -24,6 +24,7 @@
             else:
                 self.right.insert(val)
 
+
 a = BST(1)
 a.insert(2)
 a.insert(3)
```